framework/src/base/Assembly.C

```diff
@@ -915,29 +923,34 @@ Assembly::init()
   }
 
   _sub_Kee.resize(n_vars);
+  _sub_Kenl.resize(n_vars);
```

Comment:
I think ee stands for element-element, en for element-neighbor and so on. I suggest calling the new variable Keg for element-global (and 🍺 ).